test/spec/controllers/eventDetails.spec.js

```diff
@@ -268,16 +268,50 @@ describe('Controller: eventDetails', function () {
       }),
     );
 
-    it('saveEvent() should validate the Ministry Purpose', () => {
-      scope.saveEvent();
+    describe('saveEvent', () => {
+      it('should validate the Ministry Purpose', () => {
+        scope.saveEvent();
 
-      expect(scope.notify.message.toString()).toContain(
-        'Please enter Ministry Purpose.',
-      );
+        expect(scope.notify.message.toString()).toContain(
+          'Please enter Ministry Purpose.',
+        );
 
-      expect(scope.notify.message.toString()).not.toContain(
-        'Please enter which Event Type',
-      );
+        expect(scope.notify.message.toString()).not.toContain(
+          'Please enter which Event Type',
+        );
+      });
+
+      it('should validate the Event Name', () => {
```

Comment:
We might have these tests, but if not we should add them. Test to make sure the error messages do not show when there is no `&` or `"` in the name.

Reply:
Good idea, I'll add them.